src/main/java/me/cortex/nvidium/NvidiumWorldRenderer.java

```diff
@@ -104,13 +104,13 @@ public void uploadBuildResult(ChunkBuildOutput buildOutput) {
     }
 
     public void addDebugInfo(ArrayList<String> debugInfo) {
-        debugInfo.add("Using nvidium renderer: "+ Nvidium.MOD_VERSION);
-        debugInfo.add("Memory limit: " + max_geometry_memory + " mb");
+        debugInfo.add("Using Nvidium renderer: "+ Nvidium.MOD_VERSION);
+        debugInfo.add("Memory limit: " + max_geometry_memory + " MB");
         debugInfo.add("Terrain Memory MB: " + sectionManager.terrainAreana.getAllocatedMB()+(Nvidium.SUPPORTS_PERSISTENT_SPARSE_ADDRESSABLE_BUFFER?"":" (fallback mode)"));
         debugInfo.add(String.format("Fragmentation: %.2f", sectionManager.terrainAreana.getFragmentation()*100));
         debugInfo.add("Regions: " + sectionManager.getRegionManager().regionCount() + "/" + sectionManager.getRegionManager().maxRegions());
         if (asyncChunkTracker != null) {
-            debugInfo.add("Async BFS iteration time: " + asyncChunkTracker.getIterationTime());
+            debugInfo.add("ASync BFS iteration time: " + asyncChunkTracker.getIterationTime());
```

Comment:
"Async" was right actually.

Reply:
No this is a false comparison. VSync stands for "vertical synchronization" (2 words) whereas async stands for "asynchronous" (1 word).